subsys/nrf_security/src/drivers/cracen/cracenpsa/src/common.h

```diff
@@ -307,6 +331,35 @@ int cracen_get_rnd_in_range(const uint8_t *n, size_t nsz, uint8_t *out);
 psa_status_t cracen_ecc_reduce_p256(const uint8_t *input, size_t input_size, uint8_t *output,
 				    size_t output_size);
 
+/**
+ * @brief Checks if the value is a quadratic residue modulo p,
+ *	  where p is an EC prime.
+ *
+ * @param[in] curve_prime EC prime
+ * @param[in] value       Value to check
+ * @param[out] is_qr      Check result: true if value is a quadratic residue,
+ *			  false otherwise
```

Comment:
```suggestion
 * @param[in] curve_prime EC prime.
 * @param[in] value       Value to check.
 * @param[out] is_qr      Result of the check. This outputs true if the value is a quadratic residue;
 *			  false otherwise.
```